tests/test_models.py

```diff
@@ -292,8 +292,7 @@ def test_schema():
     assert tms.schema_json()
     assert tms.dict(exclude_none=True)
     json_doc = json.loads(tms.json(exclude_none=True))
-    # We cannot translate PROJ4 to epsg so it's set to None
-    assert json_doc["supportedCRS"] == "http://www.opengis.net/def/crs/EPSG/0/None"
+    assert json_doc["supportedCRS"] == "http://www.opengis.net/def/crs/IAU/2015/49930"
```

Comment:
okay I can look into that, is there a way to list out all CRSs proj has/all epsg codes?, need to set up a local dev environment to,

Reply:
ahh found https://pyproj4.github.io/pyproj/stable/api/database.html#pyproj.database.get_codes

however I seem to keep running into issues with setting up pre-commit on macos (typed-ast compilation errors) so this may take a few more commits than is ideal

Reply:
I said `all` but it's ok to test only few :D

Reply:
do not worry about pre-commit I'll fix formatting later

Reply:
okay minimal test added to the end of test_models.py